src/views/Team/Outlet.vue

```diff
@@ -2,97 +2,189 @@
+                  </li>
+                </inline-link-list>
+              </div>
+            </ion-col>
+          </div>
+
+          <ion-toolbar>
+            <!-- Scrollable Segment -->
+            <ion-segment
+              scrollable
+              value="about"
+              mode="md"
+              @ionChange="segmentChanged($event)"
+            >
+              <ion-segment-button value="qrcode">
+                <ion-icon :icon="qrCodeIcon" />
+              </ion-segment-button>
+              <ion-segment-button value="about">
+                <ion-label>About</ion-label>
+              </ion-segment-button>
+              <ion-segment-button value="news">
+                <ion-label>News</ion-label>
+              </ion-segment-button>
+              <ion-segment-button value="feed">
+                <ion-label>Feed</ion-label>
+              </ion-segment-button>
+            </ion-segment>
+          </ion-toolbar>
+
+          <div style="display: block; overflow-y: auto; height: 60vh">
+            <!-- Div About -->
+            <div v-if="state == 'about'">
               <div data-cy="description">
-                <render-md  adminMd :source="info" />
-                <ion-button data-cy-role="editModal" v-if="canEdit" color="light" @click="intendEditInfo" size="small" fill="clear">
-                  Edit Text
-                </ion-button>
+                <h2 data-cy="title" class="subTitle">
+                  {{ team.name }}
+                </h2>
+                <p>
+                  {{ team.info }}
+                  <ion-button
+                    data-cy-role="editModal"
+                    v-if="canEdit"
+                    color="primary"
+                    @click="intendEditInfo"
+                    size="small"
+                    fill="clear"
+                  >
+                    Edit Text
+                  </ion-button>
+                </p>
+                <render-md adminMd :source="info" />
               </div>
-            </div>
-            <div class="extra-actions" v-if="canEdit">
-              <ion-chip title="remove background" v-if="team.background" @click="removeBackground">
-                <ion-icon :icon="imageIcon" />
-                <ion-icon  :icon="trashIcon" />
-              </ion-chip>
-              <ion-chip v-else @click="selectBackground" title="upload background">
-                <ion-icon :icon="imageIcon" />
-                <ion-icon :icon="uploadIcon" />
-              </ion-chip>
+
+              <h2>Subteams</h2>
+              <ul v-if="subteams" class="subteams" data-cy="subteams">
+                <li v-for="t in subteams" :key="t.id">
+                  <router-link
+                    :to="{ name: 'ViewTeam', params: { teamSlug: t.slug } }"
+                  >
+                    <ion-chip outline>
+                      <avatar :profile="t" size="1.5em" />
+                      <div style="margin: 5px">{{ t.name }}</div>
```

Comment:
avatar already provides a `with-name` feature:

```suggestion
                      <avatar :profile="t" size="1.5em" with-name />
```